qutebrowser/browser/network/networkmanager.py

```diff
@@ -241,12 +241,21 @@ def clear_rejected_ssl_errors(self, url):
     @pyqtSlot('QNetworkReply', 'QAuthenticator')
     def on_authentication_required(self, reply, authenticator):
         """Called when a website needs authentication."""
-        answer = self._ask("Username ({}):".format(authenticator.realm()),
+        import netrc
+        net = netrc.netrc()
+        authenticators = net.authenticators(reply.url().host())
+        user, password = None, None
+
+        if authenticators:
```

Comment:
This should probably be `if authenticators is not None:` - better to be explicit about `None`.